ports/tinyfiledialogs/portfile.cmake

```
@@ -2,25 +2,28 @@ vcpkg_fail_port_install(ON_TARGET "uwp")
 
 vcpkg_check_linkage(ONLY_STATIC_LIBRARY)
 
-vcpkg_from_sourceforge(
-    OUT_SOURCE_PATH SOURCE_PATH
-    REPO tinyfiledialogs
-    FILENAME "frozen_versions/tinyfiledialogs-3.6.3.zip"
-    SHA512 42c3bd34b0287cf2477f9ede049bea29a9306304e8fab7740065957d3737f4041899f26f29a0693e801cb0a7b63844509f86441262303ff0a4030a431ffac648
+set(TINYFILEDIALOGS_VERSION "3.8.7")
+set(SOURCE_PATH "${CURRENT_BUILDTREES_DIR}/src/tinyfiledialogs-${TINYFILEDIALOGS_VERSION}")
+
+vcpkg_from_git(
+    OUT_SOURCE_PATH "${SOURCE_PATH}"
```

```suggestion
    OUT_SOURCE_PATH SOURCE_PATH
```
This parameter is a variable name that will be set to the location of the sources (an "output variable")